WordPress/Classes/Services/MediaService.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+
+extension MediaService {
+
+    func refreshMediaStatus(onCompletion: (() -> Void)?, onError: ((Error) -> Void)?) {
```

Comment:
If you set defaults for these parameters (`= nil`), then we can just omit them from the call site: `service.refreshMediaStatus()`.